templates/indieauth-settings.php

```diff
@@ -3,70 +3,78 @@
 					</th>
 					<td>
 						<p>
-							<?php _e( 'Authorization Endpoint:', 'indieauth' ); ?><br />
-							<code><?php echo indieauth_get_authorization_endpoint(); ?></code>
+							<?php esc_html_e( 'Authorization Endpoint:', 'indieauth' ); ?><br />
+							<code><?php echo esc_url( indieauth_get_authorization_endpoint() ); ?></code>
 						</p>
 						<p>
-							<?php _e( 'Token Endpoint:', 'indieauth' ); ?><br />
-							<code><?php echo indieauth_get_token_endpoint(); ?></code>
+							<?php esc_html_e( 'Token Endpoint:', 'indieauth' ); ?><br />
+							<code><?php echo esc_url( indieauth_get_token_endpoint() ); ?></code>
 						</p>
 					</td>
 				</tr>
 				<tr>
 					<th>
-						<?php _e( 'Set User to Represent Site URL', 'indieauth' ); ?>
+						<?php esc_html_e( 'Set User to Represent Site URL', 'indieauth' ); ?>
 					</th>
 					<td>
 						<label for="indieauth_root_user">
-							<?php wp_dropdown_users(
+							<?php
+							wp_dropdown_users(
 								array(
 									'show_option_all' => __( 'None', 'indieauth' ),
-									'name' => 'indieauth_root_user',
-									'id' => 'indieauth_root_user',
-									'show' => 'display_name_with_login',
-									'selected' => get_option( 'indieauth_root_user' )
+									'name'            => 'indieauth_root_user',
+									'id'              => 'indieauth_root_user',
+									'show'            => 'display_name_with_login',
+									'selected'        => get_option( 'indieauth_root_user' ),
 								)
-							); ?>
-							<?php _e( 'Set a User who will represent the URL of the site', 'indieauth' ); ?>
+							);
+							?>
+							<?php esc_html_e( 'Set a User who will represent the URL of the site', 'indieauth' ); ?>
 						</label>
 					</td>
 				</tr>
 			</tbody>
 		</table>
 
-		<h2 class="title"><?php _e( 'Web Sign-In', 'indieauth' ); ?></h2>
+		<h2 class="title"><?php esc_html_e( 'Web Sign-In', 'indieauth' ); ?></h2>
 
-		<p><?php _e( 'Enable Web Sign-In for your blog, so others can use IndieAuth or RelMeAuth to log into this site.', 'indieauth' ); ?></p>
+		<p><?php esc_html_e( 'Enable Web Sign-In for your blog, so others can use IndieAuth or RelMeAuth to log into this site.', 'indieauth' ); ?></p>
 
 		<table class="form-table">
 			<tbody>
 				<tr>
 					<th>
-						<?php _e( 'Use IndieAuth login', 'indieauth' ); ?>
+						<?php esc_html_e( 'Use IndieAuth login', 'indieauth' ); ?>
 					</th>
 					<td>
 						<label for="indieauth_show_login_form">
-							<input type="checkbox" name="indieauth_show_login_form" id="indieauth_show_login_form" value="1" <?php
-								echo checked( true, get_option( 'indieauth_show_login_form' ) );  ?> />
-							<?php _e( 'Add a link to the login form to authenticate using an IndieAuth endpoint.', 'indieauth' ); ?>
+							<input type="checkbox" name="indieauth_show_login_form" id="indieauth_show_login_form" value="1" 
+							<?php
+								echo checked( true, get_option( 'indieauth_show_login_form' ) );
```

Comment:
Nit but this could also just be:
```suggestion
								echo checked( get_option( 'indieauth_show_login_form' ) );
```

Reply:
Actually this should be just 
```suggestion
								checked( get_option( 'indieauth_show_login_form' ) );
```
since the function is defined as `checked( mixed $checked, mixed $current = true, bool $echo = true )`